paper.md

```diff
@@ -0,0 +1,91 @@
+    orcid: XX
+    affiliation: 1
+  - name: Anthime Bucquet
+    orcid: XX
+    affiliation: 1
+  - name: Joseph Paul Cohen
+    orcid: XX
+    affiliation: "2, 3"
+  - name: Julien Cohen-Adad
+    orcid: 0000-0003-3662-9532
+    affiliation: "1, 4"
+affiliations:
+ - name: NeuroPoly Lab, Institute of Biomedical Engineering, Polytechnique Montreal, Montreal, Canada
+   index: 1
+ - name: AIMI, Stanford University, Stanford, CA, USA
+   index: 2
+ - name: Mila, Quebec AI Institute, Montreal, QC, Canada
+   index: 3
+ - name:  Functional Neuroimaging Unit, CRIUGM, Université de Montréal, Montreal, QC, Canada
+   index: 4
+date: 3 November 2020
+bibliography: paper.bib
+
+---
+
+# Summary
+
+`ivadomed` is an open-source Python package for designing, end-to-end training, and evaluating deep learning models applied to medical imaging data. The package includes APIs, command-line tools, documentation, and tutorials. `ivadomed` also includes pre-trained models such as spinal tumor segmentation and vertebral labeling. Original features of `ivadomed` include a data loader that can parse image metadata (e.g., acquisition parameters, image contrast, resolution) and subject metadata (e.g., pathology, age, sex) for custom data splitting or extra information during training and evaluation. Any dataset following the [Brain Imaging Data Structure (BIDS)](https://bids.neuroimaging.io/) convention will be compatible with `ivadomed` without the need to manually organize the data, which is typically a tedious task. Beyond the traditional deep learning methods, `ivadomed` features cutting-edge architectures, such as FiLM [@perez2017film] and HeMis [@havaei2016hemis], as well as various uncertainty estimation methods (aleatoric and epistemic), and losses adapted to imbalanced classes and non-binary predictions. Each step is conveniently configurable via a single file. At the same time, the code is highly modular to allow addition/modification of an architecture or pre/post-processing steps. Example applications of `ivadomed` include MRI object detection, segmentation, and labeling of anatomical and pathological structures. Overall, `ivadomed` enables easy and quick exploration of the latest advances in deep learning for medical imaging applications. `ivadomed`'s main project page is available at https://ivadomed.org.
+
+# Statement of need
+
+Deep learning is increasingly used in medical image processing [@kim_deep_2019]. It provides automated solutions to repetitive and/or tedious tasks such as the segmentation of pathological structures. However, medical imaging data present many challenges: datasets are often not publicly-available, ground truth labels are scarce due to the limited availability of expert raters, and needs can be very specific and tailored to particular datasets (e.g., segmentation of spinal tumors on sagittal MRI T2-weighted scans). Thus, offering solution for convenient training models (or fine-tuning of pre-existing models) is needed.
```

Comment:
```suggestion
Deep learning is increasingly used in medical image processing [@kim_deep_2019]. It provides automated solutions to repetitive and/or tedious tasks such as the segmentation of pathological structures. However, medical imaging data present many challenges: datasets are often not publicly-available, ground truth labels are scarce due to the limited availability of expert raters, and needs can be very specific and tailored to particular datasets and tasks. There already exists a variety of deep learning framework for medical imaging, but [each have their pros & cons](https://ivadomed.org/en/latest/purpose.html#comparison-with-other-projects). 
```

Reply:
I am a bit hesitant about integrating this one: we would be highlighting that (unfortunately) we are not the only one, without developing our strengths.

Reply:
I think we need to mention it. At this point, there are already >15 similar toolboxes and ignoring them would be dishonest and would give the impression that we developed this toolbox in our bubble. If readers get that impression, this they will judge ivadomed very negatively. If we demonstrate we are aware of the state-of-the-art, it will give the impression that we can "only do better", because we know what's out there.  I'll find a rephrasing to spin ivadomed's contribution more positively.

Reply:
Mmmm I agree! THanks for explaining.
I agree even more when reading the [JOSS checklist](https://joss.readthedocs.io/en/latest/review_checklist.html#software-paper):
```
State of the field: Do the authors describe how this software compares to other commonly-used packages?
```